components/org.apache.stratos.python.cli/src/main/python/cli/Configs.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+stratos_prompt = "stratos> "
+
+stratos_dir = "~/.stratos"
+log_file_name = "stratos-cli.log"
+
+stratos_dir_path = os.path.expanduser(stratos_dir)
+log_file_path = os.path.join(stratos_dir_path, log_file_name)
+
+stratos_url = os.getenv('STRATOS_URL', "https://localhost:9443/")
```

Comment:
Are these configurable options by the user? If they are there shouldn't be code here. Better go for a config file such as an *.ini file and read from it.